src/filefinder.cpp

```diff
@@ -191,30 +195,49 @@ namespace {
 					}
 				}
 			}
-			return name;
+			return std::string();
 		}
 		return file_it->second;
 	}
 
 	std::string FindFile(const std::string &dir, const std::string& name, const char* exts[]) {
+		std::string rtp_name;
+
 		const std::shared_ptr<FileFinder::DirectoryTree> tree = FileFinder::GetDirectoryTree();
 		std::string const ret = FindFile(*tree, dir, name, exts);
 		if (!ret.empty()) { return ret; }
 
-		const std::string& rtp_name = translate_rtp(dir, name);
+		// Try RTP if enabled and available
+		if (!disable_rtp)
+			rtp_name = translate_rtp(dir, name);
+
+		if (!rtp_name.empty()) {
+			// RPG_RT will even load RTP files when the game disables it
+			if(disable_rtp_warnings && !warning_broken_rtp_game_shown) {
+				warning_broken_rtp_game_shown = true;
+				Output::Warning("This game claims it does not need the RTP, but actually uses files from it!");
```

Comment:
Though this is nice to be shown and helps a lot users, there are a significant amount of games with missing audio files and this could annoy users as we had before. I think this should be fine with non-audio related assets, but output them as debug for audio.

Reply:
Yeah, I was unsure and thought about making this `Debug` only as well.